### PR TITLE
Update various template files to increase accessibility support.

### PR DESCRIPTION
**Pull request by mookman288**
Hi, 

I read the guidelines, but unfortunately your PHP Unit configuration is designed for unix, rather than system agnostic, so I won't be able to automate tests without significant changes (which seem unreasonable for the PR.) That being said, at the very least if you do not adopt this PR, you can see all of the places where I found icon tags that didn't have the proper accessibility support. 

As it wasn't part of the guidelines of the issue, I didn't implement aria attributes. However, I would recommend them. 

Hope that helps!

### CODE REVIEW

File: templates/part.board.mainView.php [hunk 1]
```diff
@@ -114,7 +114,7 @@ class="icon icon-delete"></span><span><?php p($l->t('Delete')); ?></span></a>
 							   required placeholder="<?php p($l->t('Enter a card title')); ?>"/>
 					</h4>
 				</form>
-				<div ng-if="!status.addCard[s.id]" ng-click="status.addCard[s.id]=true">
+				<div ng-if="!status.addCard[s.id]" ng-click="status.addCard[s.id]=true" title="<?php p($l->t('Add card')); ?>">
```
Comment (juliushaertl) [inline templates/part.board.mainView.php]: The span.hidden-visually is missing here

**Comment by juliushaertl** [comment]
On the tests,works fine since this is a template only change. Contributions to make the PHPunit testing work cross platform are of course always welcome. 😉

**Comment by juliushaertl** [comment]
This is a fix for https://github.com/nextcloud/deck/issues/149

File: templates/part.boardlist.php [hunk 2]
```diff
@@ -70,16 +71,16 @@
...
 				</div>
 				<div class="app-popover-menu-utils" ng-if="b.deletedAt > 0">
-					<button class="icon icon-history button-inline" ng-click="boardDeleteUndo(b)" title="Undo board deletion - Otherwise the board will be deleted during the next cronjob run."></button>
+					<button class="icon icon-history button-inline" ng-click="boardDeleteUndo(b)" title="Undo board deletion - Otherwise the board will be deleted during the next cronjob run."><span class="hidden-visually"><?php p($l->t('Undo board deletion - Otherwise the board will be deleted during the next cronjob run.')); ?></span></button>
 				</div>
 			</td>
 		</tr>
 		<tr ng-if="status.filter === '' && !status.addBoard" ng-click="status.addBoard=!status.addBoard" class="board-create">
-			<td><span class="icon icon-add"></span></td>
+			<td><span class="icon icon-add" title=""></span></td>
```
[the remark this answers is not in the document]
Comment (mookman288) [reply templates/part.boardlist.php]: You're right. No point in having title tags here because the input itself is enough.

**Comment by mookman288** [comment]
New commits made. Please review.

File: templates/part.boardlist.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (juliushaertl) [reply templates/part.boardlist.php]: Can you also remove them? :wink:

**Comment by mookman288** [comment]
@juliushaertl An empty alt tag tends to indicate to the screen reader to ignore it, so I made the mistake of assuming the same for the title tag. Apologies. I've made that change.

**Comment by juliushaertl** [comment]
@mookman288 Can you signoff the 80b38b9  commit as well, then our CI will also be happy :)

**Comment by juliushaertl** [comment]
Let's merge to get this in. The change is rather small.

**Pull request merged**